modules/dataquery/jsx/react.sidebar.js

```diff
@@ -1,66 +1,87 @@
-var Sidebar = React.createClass({
-    getInitialState: function() {
-        return {
-            'hidden' : false
-        };
-    },
-    toggleHidden: function() {
-        this.setState({ 'hidden' : !this.state.hidden });
-    },
-    hide: function() {
-        this.setState({ 'hidden' : true });
-    },
-    show: function() {
-        this.setState({ 'hidden' : false});
-    },
-    render: function() {
-        if(this.state.hidden) {
-            return <div />;
-        }
-        return (
-                <div>
-                    <h2>{this.props.Name}</h2>
-                    {this.props.children}
-                </div>
-       );
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class Sidebar extends Component {
```

Comment:
im confused as to why all these React components live inside the data query module directory

Reply:
@zaliqarosli The data query module uses them and they seem to be custom. I think leaving cleaning up that to a different PR is best. Also the DQT is in status of being redesigned.